package.json

```diff
@@ -1,10 +1,7 @@
 {
   "private": true,
-  "napa": {
-    "github-fork-ribbon-css": "simonwhitaker/github-fork-ribbon-css"
-  },
   "scripts": {
-    "start": "NODE_ENV=production node server",
+    "start": "NODE_ENV=production node-dev server",
```

Comment:
`npm start` is what the infrastructure will use to start the project during deployment. We should not be using `node-dev` in those settings. That's what `dev-server` task was for.